index.html

```diff
@@ -0,0 +1,255 @@
+                </div>
+            </div>
+
+            <!-- 3RD WORK SECTION -->
+            <div class="work-box">
+                <div class="work-img">
+                    <img src="asset/Snapshoot-Portfolio1.svg" alt="work_img1">
+                </div>
+                <div class="work-info">
+                    <h2 class="work-title">Tonic</h2>
+                    <div class="work-meta">
+                        <p class="work-para">canopy</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para1">Back End Dev</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para2">2015</p>
+                    </div>
+                    <div class="work-note">
+                        <p>A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required.</p>
+                    </div>
+                    <div class="work-tag">
+                        <ul>
+                            <li> <div class="tag">html</div></li>
+                            <li> <div class="tag1">css</div></li>
+                            <li> <div class="tag2">javascript</div></li>
+                        </ul>
+                    </div>
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </div>
+
+            <!-- 4TH WORK SECTION -->
+            <div class="work-box1">
+                <div class="work-img">
+                    <img src="asset/Snapshoot-Portfolio2.svg" alt="work_img1">
+                </div>
+                <div class="work-info">
+                    <h3 class="work-title">Multi-Post Stories</h3>
+                    <div class="work-meta">
+                        <p class="work-para">canopy</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para1">Back End Dev</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para2">2015</p>
+                    </div>
+                    <div class="work-note">
+                        <p>A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required.</p>
+                    </div>
+                    <div class="work-tag">
+                        <ul>
+                            <li> <div class="tag">html</div></li>
+                            <li> <div class="tag1">css</div></li>
+                            <li> <div class="tag2">javascript</div></li>
+                        </ul>
+                    </div>
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- MY ABOUT SECTION -->
+<section class="about-section">
```

Comment:
- [ ] This section has a fixed padding of `5.5rem` causing it break on small phone screens. kindly fix it so it's identical to the design.
![image](https://user-images.githubusercontent.com/33767951/164948978-4595b029-465e-467e-a04e-7f0b669e5169.png)